db/driver_inmem.go

```diff
@@ -508,7 +508,7 @@ func (a *ADMInMem) UpdateLiveSessionPersistence(sc ot.SpanContext, lsID string,
 		return err
 	}
 	a.liveSessions[lsID].Persistent = persistent
-	return nil
+	return a.liveSessions[lsID]
```

Comment:
Please revert this change; we don't need/want to return a whole liveSession from an update function. These are just meant to return a simple error that the caller can check.

Reply:
ok, will update the code